graylog2-server/src/main/java/org/graylog/plugins/views/storage/migration/state/actions/MigrationActions.java

```diff
@@ -55,10 +55,6 @@ public interface MigrationActions {
 
     boolean dataNodeStartupFinished();
 
-    void setStateMachineContext(MigrationStateMachineContext context);
```

Comment:
unused import of `MigrationStateMachineContext` should be removed

Reply:
removed, thx!